src/main/java/org/prebid/server/metric/Metrics.java

```diff
@@ -303,4 +303,12 @@ public void updateStoredImpsMetric(boolean found) {
             incCounter(MetricName.stored_imps_missing);
         }
     }
+
+    public void updateStoredCacheRequestSuccessTime(long timeElapsed) {
```

Comment:
Pls rename to `updateCacheRequestSuccessTime`.
PS. Check the same for `updateStoredCacheRequestFailedTime`.